messages/views.py

```diff
@@ -107,28 +110,33 @@ def message(request, message_id):
     tvars = {'message': message}
     return render(request, 'messages/message.html', tvars)
 
+
 @login_required
 @transaction.atomic()
 def new_message(request, username=None, message_id=None):
+
+    if request.user.profile.num_sounds > 0 or request.user.profile.num_posts > 5:
```

Comment:
I thought about it but then I saw it would not be re-used (by now). The thing is that different situations would require different levels of trustworthiness. For example, to decide whether or not to display the user's about text in profile we check for number of downloads. This is a less strict trustworthiness measure. I would be happy to unify different places where similar things are done and maybe have a `is_trustworthy` method in `Profile` with some sort of `level` parameter to decide how strict we are. This requires some investigation to see in which places this method could be used. Maybe I could implement the method only for this case and open an issue for the further investigations...

Reply:
Added in 83a8a84ae96bee7da050771ca70019ba20d74a9c